components/centraldashboard/public/assets/i18n/languages.json

```diff
@@ -170,6 +175,17 @@
             "notebookCard.errRetrievingNotebooks": "Erreur en récupérant les bloc-notes",
             "notebookCard.errNoNotebooks": "Pas de bloc-notes dans l'espace de noms {namespace}",
             "notebookCard.txtAccessed": "Accédé ",
+            "notebookDefaultCard.connect": "Connecter",
+            "notebookDefaultCard.create": "Créer",
+            "notebookDefaultCard.details": "Détails",
+            "notebookDefaultCard.name": "Nom",
+            "notebookDefaultCard.type": "Type",
+            "notebookDefaultCard.helpText": "Le bloc-note par défaut est celui qui démarre automatiquement lors de la connexion à Kubeflow",
+            "notebookDefaultCard.txtDefaultNotebook": "Bloc-note par défaut",
+            "notebookDefaultCard.txtDataLoading": "Les données sont en cours de chargement",
+            "notebookDefaultCard.txtNoDefaultNotebook": "Le bloc-note par défaut est celui qui démarrera automatiquement lors de la connexion à Kubeflow. Si vous voyez cette note, le système n'a pas détecté de notebook par défaut pour cet espace de noms. Vous pouvez cliquer sur 'Créer' ci-dessous pour en générer un. .",
```

Comment:
"Notebook par defaut" should be translated. Also, last sentence ends with 2 dots.